### PR TITLE
Update "hello world" example notebook to TF v2.2.0

### PR DESCRIPTION
**Pull request by carloshvp**
I started these changes because the notebook for training and creating the model fails to run in Colab out-of-the-box, as gast dependency is not compatible with tensorflow-probability. This is the error observed when running the cell with `! pip install -q tensorflow==2`

> ERROR: tensorflow-probability 0.10.0 has requirement gast>=0.3.2, but you'll have gast 0.2.2 which is incompatible.

Simply by upgrading to TF v2.2.0 everything works again without any other workarounds

### CODE REVIEW

**Review by shivaylamba — APPROVED** [review]

**Comment by lc0** [comment]
Colab, would have by default have as of now version 2.2.0 

So might be helpful to skip pip install and just validate current version of TF

**Comment by lc0** [comment]
Somehow I can not see how to get quantisation back working, even with proper 
```python
converter.target_spec = tf.lite.TargetSpec(supported_types=[tf.int8])
```

@sayakpaul do you have any idea? :)

**Comment by sayakpaul** [comment]
@lc0 could you leave a link to the notebook?

**Comment by lc0** [comment]
> @lc0 could you leave a link to the notebook?

https://colab.research.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/examples/hello_world/train/train_hello_world_model.ipynb

**Comment by sayakpaul** [comment]
@lc0 I tried with the following:

```python
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
converter.inference_input_type = tf.uint8
converter.inference_output_type = tf.uint8
```

But weirdly, it resulted in:

```
Model is 2540 bytes
Quantized model is 2560 bytes
Difference is -20 bytes
```

@khanhlvg could you shed some light?

**Comment by khanhlvg** [comment]
The model itself is very small with just a handful number of weights. So using quantization won't save you much space. However, quantization adds some overhead to the model size (e.g. a map between quantized values and float value) which result in the small difference that you see.

**Comment by lc0** [comment]
@khanhlvg @sayakpaul That's also what I tried and got the same result.

Btw, it looks like the previous version was able to reduce file size. Did we change how much overhead it requires? 
Also, I tested with saved model vs keras. And it seems like saved models generates bigger TFLite file, than the same Keras model.

Thanks for your explanations!

**Comment by gbaned** [comment]
@carloshvp Can you please resolve conflicts? Thanks!

**Comment by gbaned** [comment]
@carloshvp Any update on this PR? Please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
